### PR TITLE
Templates

### PR DESCRIPTION
**Pull request by Csasaka19**
Parents section in the admin panel .
The string representation of user is now the first name and last name of the Account user.

### CODE REVIEW

File: daystar_orientation/account/admin.py [hunk 1]
```diff
@@ -9,13 +9,13 @@ class AccountAdmin(UserAdmin):
...
     fieldsets = UserAdmin.fieldsets + (
-        (None, {'fields': ('user_type', 'phone_number', 'admission_number', 'course', 'campus', 'gender', 'accomodation')}),
+        (None, {'fields': ('user_type', 'phone_number', 'admission_number', 'course', 'campus', 'gender', 'accomodation', 'parent')}),
     )
 
     add_fieldsets = (
         (None, {
             'classes': ('wide',),
-            'fields': ('admission_number', 'email', 'first_name', 'last_name', 'username', 'campus', 'course', 'phone_number', 'user_type', 'password1', 'password2', 'gender', 'accomodation', 'checked_in'),
+            'fields': ('admission_number', 'email', 'first_name', 'last_name', 'username', 'campus', 'course', 'phone_number', 'user_type', 'password1', 'password2', 'parent','gender', 'accomodation', 'checked_in'),
```
Comment (eiidoubleyuwes) [inline daystar_orientation/account/admin.py:18]: Would this necessitate the drop of the DB?As from testing on your end did you have to delete the database in order to update the tables?

File: daystar_orientation/account/views.py [hunk 2]
```diff
@@ -78,7 +78,7 @@ def post(self, request, *args, **kwargs):
                 'campus': user.campus,
                 'email': user.email,
                 'admission_number': user.admission_number,
-                'course': user.course,
+                'course': user.course.name,
```
Comment (eiidoubleyuwes) [inline daystar_orientation/account/views.py:81]: Looks good

**Review by eiidoubleyuwes — COMMENTED** [review]
From your local testing,do you need to drop the DB in order for these changes to come into effect?

File: daystar_orientation/account/views.py (hunk 2 above)
Comment (Csasaka19) [reply daystar_orientation/account/views.py:81]: Nope no data is affected

**Pull request merged**